Gemfile

```diff
@@ -7,6 +7,15 @@ gem 'net-imap' # for rspec
 gem 'net-pop'  # for rspec
 gem 'net-smtp' # to send email
 
+gem 'activestorage', '>= 6.1.7.7'
```

Comment:
* Added explicit upgrades to Gemfile.
* Seems like we might want to bump Rails to 7 at some point, not now though.

Reply:
Its probably better to say ~> 6.1 or ~> 6.1.7.7

This would stop an unintentional upgrade to 7.